data/battle_anim_scripts.s

```diff
@@ -23251,16 +23251,21 @@ Move_SKY_UPPERCUT:
 	end
 
 Move_SECRET_POWER:
```

Comment:
should these animations be updated to [Gen 7](https://bulbapedia.bulbagarden.net/wiki/Secret_Power_%28move%29#Generation_VII)?